static/graph.js

```diff
@@ -217,6 +220,5 @@ const zoomHandler = d3
   .on("zoom", resize);
 
 zoomHandler(svg);
-restart();
```

Comment:
Probably a stupid question, but where did that go?

Reply:
So right below this, on the line that says `vscode.postMessage({ type: "ready" });`, that indicates to the extension side of things that the webview is ready to receive data. Then the extension will send a `refresh` message to the webview with the data.

The only point of it is to just skip the initial render where there is no data. This way, you don't have to do the thing where you stringify and paste in the initial data into the HTML directly.